run_sorting_hat_test_data.sh

```diff
@@ -21,6 +56,13 @@ CONDA_PREFIX_LINE="--conda-prefix $SN_CONDA_PREFIX"
 export LOG_PATH=${LOG_PATH:-$( pwd )/sorting.log}
 USUAL_SM_ERR_OUT=${USUAL_SM_ERR_OUT:-$( pwd )/snakemake.log}
 
+# isl db
+ORACLE_BASE=
+ORACLE_HOME=
+PYTHON_USER=
+PYTHON_CONNECT_STRING=
+PYTHON_PASSWORD=
```

Comment:
If this is meant to be like a sample script and the one with all the real values is stored elsewhere, maybe we can then rename this to `*.sample.sh`.

Reply:
Otherwise I would suggest that this script can read these vars from outside and then complain if not set.